src/Grid/grid_utils.jl

```diff
@@ -0,0 +1,21 @@
+
+
+"""
+    compute_bounding_box(grid::AbstractGrid)
+
+Computes the bounding box for a given grid, based on its node coordinates. 
+Returns the minimum and maximum vertices of the bounding box.
+"""
+function compute_bounding_box(grid::AbstractGrid{dim}) where {dim}
+    T = get_coordinate_eltype(grid)
+    min_vertex = Vec{dim}(i->T(+Inf))
+    max_vertex = Vec{dim}(i->T(-Inf))
+    for node in getnodes(grid)
```

Comment:
Might be useful to compute the bounding box for a subset, or a faceset etc, but can be added later.